go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/openbaya/prevue
+
+go 1.16
```

Comment:
What is the latest stable version of go? If it's different can you please check the differences so that we know which version is good to use?

Reply:
Sure Madhu.